test/commons/dom/url-props-from-attribute.js

```diff
@@ -55,19 +55,9 @@ describe('dom.urlPropsFromAttribute', function() {
 			vNode.actualNode,
 			'href'
 		);
-		/**
-		 * Note:
-		 * Given `a` has empty `href`, the url props will depend on the context in which the page is loaded
-		 * eg:
-		 * 	- http://localhost:9876/test/commons
-		 *  - http://localhost:9876/test/commons/?grep=url...
-		 *
-		 * Hence only asserting the what we know to be static values
-		 */
-		assert.equal(actual.protocol, 'http:');
-		assert.equal(actual.hostname, 'localhost');
-		assert.equal(actual.port, '9876');
-		assert.equal(actual.pathname, '/test/commons/');
+		assert.equal(actual.protocol, location.protocol);
```

Comment:
Since we can now longer guess what the port / hostname will be, we should get them from the current location. This is actually more correct as when using Parallels to test this file my hostname would be `192.168.1.n` and would fail this test.